src/components/common/applicant-info-2.vue

```diff
@@ -179,7 +179,7 @@
             <template v-slot:activator="{ on }">
               <div v-on="on">
                 <v-checkbox
-                  v-if="showPriorityRequest"
+                  v-if="getShowPriorityRequest"
```

Comment:
Don't change now, but if you get a change later, I recommend changing booleans getters to "isXXX" or "doXXX". (low priority)

Reply:
Don't change now.